packages/account/src/Components/poi/status/unsupported/card-details/uploader.tsx

```diff
@@ -13,8 +12,24 @@ const DROPZONE_ERRORS = {
     'too-many-files': localize('Please select one file only'),
     GENERAL: localize('Sorry, an error occured. Please select another file.'),
 };
+type TDROPZONE_ERRORS = Readonly<{
+    'file-too-large': string;
+    'file-invalid-type': string;
+    'too-many-files': string;
+    GENERAL: string;
+}>;
 
-const Message = ({ data, open }) => (
+type TUploader = {
+    data: FormikValues;
+    value: FormikValues;
+    is_full: boolean;
+    has_frame: boolean;
+    onChange: (e: unknown) => void;
+    setFieldValue: FormikProps<FormikValues>['setFieldValue'];
+    handleChange: (file: object | null, setFieldValue?: FormikProps<FormikValues>['setFieldValue']) => void;
+};
+
+const Message = ({ data, open }: FormikValues) => (
```

Comment:
does it work? here you define Message props as FormikValues, and on line 146 open is ()=>void